x-pack/plugins/ml/public/timeseriesexplorer/components/context_chart_mask/context_chart_mask.js

```diff
@@ -8,14 +8,14 @@
 
 import d3 from 'd3';
 
-import { drawLineChartDots } from '../util/chart_utils';
+import { drawLineChartDots } from '../../../util/chart_utils';
 
 /*
  * Creates a mask over sections of the context chart and swimlane
  * which fall outside the extent of the selection brush used for zooming.
  */
 // eslint-disable-next-line kibana-custom/no-default-export
```

Comment:
Can this comment be removed now `default` has been removed?

Reply:
👍 Done in fb52cccbf9